definitions/vertical_scaling.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Vertical Scaling
+status: Completed
+category: Concept
+---
+
+## Vertical Scaling
+
+### What it is
+
+The ability for an application to scale, typically by adding CPU and memory as the workload increases. Also known as “scale up”, this refers to adding resources to a single node.
+
+### Problem it Addresses
+
+[Scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) is essential in that it contributes to competitiveness, efficiency, reputation and quality. Vertical scaling is on one of the two kinds Vertical, [Horizontal](https://github.com/cncf/glossary/blob/main/definitions/horizontal_scaling.md) of scalability.
+
+As demand for an application grows beyond the current capacity of that application instance, we need to find a way to scale (add capacity to) the system. We can either add more nodes to the system ([horizontal scaling](https://github.com/cncf/glossary/blob/main/definitions/horizontal_scaling.md)) or more compute resources to existing nodes (vertical scaling).
+
+### How it helps
```

Comment:
This isn't talking about the problem it addresses but rather what it is. Here's my suggestion:

Vertical scaling allows an app to process more requests and do more work concurrently without adding more nodes to the system (scale horizontally).

Reply:
Thank you for proposing the changes, shall i raise a new commit regarding the proposed changes ?

Reply:
Not yet. Discussed with Jason and your definition works but needs more context. Below is an updated version (a mix of both versions). Let me review with Jason on Friday. Hoping to get this approved and merged before the weekend. 

Vertical scaling allows you to resize your server without changing the application code. That contrasts to horizontal scaling, where the app must be replicable to scale, potentially requiring code updates. Vertical scaling increases the capacity of an existing application by adding compute resources, allowing the app to process more requests and do more work concurrently.